school_library_decorator/main.rb

```diff
@@ -144,17 +40,18 @@ def main
 
     case entry
     when '1'
-      app.all_books
+      Book.all_books(app.books)
     when '2'
-      app.all_people
+      Person.all_people(app.people)
     when '3'
-      app.create_person
+      new_person = Person.create_person
+      app.people << new_person if new_person
     when '4'
-      app.create_book
+      app.books = Book.create_book
     when '5'
-      app.create_rental
+      @rental << Rental.create_rental(app.books, app.people)
     when '6'
-      app.all_rentals
+      Rental.all_rentals(app.rentals)
     when '7'
       puts 'Thank you for using the app!'
     end
```

Comment:
- [ ] Kindly refactor this piece of code to follow the Single responsibility rule